owtf.py

```diff
@@ -193,6 +193,9 @@ def process_options(user_args):
         'Args': args
     }
 
+def build_react_bundle():
+    print("\033[93m[*] Building the react bundle. Plese wait..")
+    os.system("npm run build > /dev/null")
```

Comment:
It will just build the bundle.js in /includes/src/build. Internally it will run "$ NODE_ENV=production webpack". See package.json